File: DayFour/Java/TwoSumTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class TwoSumTest{
    @Test
    public void testThatTheArrayisreturnedCorrectResult(){
        
        int [] numbers = {1, 2, 3, 4, 5};
        int target = 6;
        int [] actual = TwoSum.returnArray(numbers, target);
        int [] expected = {1, 5};
        assertArrayEquals(actual, expected);
    }
    
    @Test
    public void testThatItCanTakeMoreThanFiveNumbers(){
    
        int [] numbers = {1, 2, 5, 7, 6, 8, 9};
        int target = 12;
        int [] actual = TwoSum.returnArray(numbers, target);
        int [] expected = {5, 7};
        assertArrayEquals(actual, expected);
    
    }
}

File: DayFour/Python/two_sum_test.py
from unittest import TestCase
from two_sum import *

class TestTheSumOfNumbers(TestCase):

    def testThatTheArrayIsReturnedAndIsCorrect(self):
        
        
        numbers = {1, 2, 3, 4, 5}
        target = 6
        actual = returnArray(numbers, target)
        expected = [1, 5]
        self.assertEqual(actual, expected)
